### PR TITLE
 Replace Tastypie in export with DRF

### PR DESCRIPTION
**Pull request by praseodym**
Also includes changes from #219 and #220.

Deployment checklist:
- [ ] Execute `./manage.py migrate tastypie zero` before deployment (Tastypie needs to be in `INSTALLED_APPS`)

### CODE REVIEW

File: ldb/export.py [hunk 1]
```diff
@@ -189,44 +167,34 @@ def set_filters(self, query):
...
-        if hasattr(bundle.request, 'GET'):
-            get = bundle.request.GET.copy()
-
-        querysets = self.set_querysets(get)
+    def get(self, request, format=None):
+        querysets = self.set_querysets(request.query_params)
 
         if not querysets:
-            return HttpBadRequest("No groups selected")
+             return Response("No groups selected", status=status.HTTP_400_BAD_REQUEST)
```
Comment (jgadelange) [inline ldb/export.py]: I think the indent is wrong :P

File: ldb/urls.py [hunk 2]
```diff
@@ -40,5 +33,6 @@
     path('organizations/create/', OrganizationEditView.as_view(), name='ldb_organizations_create'),
     path('committees/', CommitteeMembershipFilterView.as_view(), name='ldb_committees'),
 
-    path('export/', ExportView.as_view(), name="ldb_export"),
+    path('export/', ExportView.as_view(), name='ldb_export'),
+    path('export/exec', export.Export.as_view(), name='ldb_export_execute'),
```
Comment (jgadelange) [inline ldb/urls.py]: It feels as if this should be the same view (one with `GET`, other `POST`) . Also, convention is to have a trailing slash in the urls.

Comment (jgadelange) [reply ldb/urls.py]: You should be able to overwrite the `get()` and `post()` in the `APIView`. I believe when you return a `HttpResponse` in the get you can return a rendered template, the post should then return what the `APIView` currently does. But maybe we should just leave that for now.

As for the trailing slash convention. Django has an [`APPEND_SLASH`](https://docs.djangoproject.com/en/dev/ref/settings/#append-slash) setting which defaults to `True`:
>When set to True, if the request URL does not match any of the patterns in the URLconf and it doesn’t end in a slash, an HTTP redirect is issued to the same URL with a slash appended.

From which I infer that having trailing slashes is the convention. But mostly it is that the rest of this project has trailing slashes, so for this project at least it is the convention.

File: ldb/export.py [hunk 3]
```diff
@@ -119,8 +103,7 @@ class Meta:
 
     def set_fields(self, query):
         allowed_fields = flatten(self.allowed_fields)
-        requested_fields = query.get("fields", "[]")
-        requested_fields = simplejson.loads(str(requested_fields))
```
Comment (praseodym) [inline ldb/export.py]: No more `simplejson` either :smiley_cat:

File: ldb/urls.py (hunk 2 above)
Comment (praseodym) [reply ldb/urls.py]: Done!

**Comment by jgadelange** [comment]
LGTM

**Pull request merged**